src/webapp/reports/autogenerated-forms/SectionsTabs.tsx

```diff
@@ -94,21 +94,20 @@ function TypeSwitch(props: TypeSwitchProps) {
     }
 }
 
-function isTabHeader(order: number | undefined) {
+function isTabHeader(order: string | undefined) {
     if (order === undefined) return false;
-    if (order % 1 === 0) return true;
 
-    if (order === Math.floor(order) + 0.1) {
-        return true;
-    }
+    const [_primaryTabIndex, secondaryTabIndex] = order.split(".");
+    if (secondaryTabIndex === undefined || secondaryTabIndex === "0" || secondaryTabIndex === "1") return true;
 }
 
 const AutoFormComponent = React.memo(TypeSwitch);
 
 const TabPanel: React.FC<TabProps> = React.memo(props => {
     const { section, dataFormInfo, value } = props;
     const { viewType, tabs } = section;
-    const index = tabs.order !== undefined ? Math.floor(tabs.order) : -1;
+    const primaryTabIndex = tabs.order?.split(".")[0];
+    const index = tabs.order !== undefined && primaryTabIndex ? parseInt(primaryTabIndex) : -1;
```

Comment:
If `tabs.order` is undefined `primaryTabIndex` will be undefined as well, right? That means we don't need to check again if `tabs.order !== undefined` in the next line.

I think we can simplify to:

`const index = primaryTabIndex ? parseInt(primaryTabIndex) : -1;`